kolibri/core/assets/src/views/CoreMenu/CoreMenuOption.vue

```diff
@@ -141,6 +145,10 @@
           this.$emit('select');
         }
       },
+      createLabel(text) {
+        const string = `${text.toLowerCase()}Label`;
+        return this.coreString(string);
```

Comment:
This creates an implicit coupling between the plugin and the core strings, so this makes this not very pluggable.